llvm/test/CodeGen/Xtensa/load.ll

```diff
@@ -0,0 +1,13 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc --mtriple=xtensa < %s | FileCheck %s
+
+define signext i8 @test_load_i8(i32 noundef %0){
```

Comment:
Why don't pass the pointer as an argument?
```suggestion
define signext i8 @test_load_i8(ptr %p){
```

Reply:
Thank you for comment. Fixed